bot/cogs/geo_cog/geo_cog.py

```diff
@@ -0,0 +1,547 @@
+        o5.callback = o5incorrect
+
+        OptionsList = list()
+        OptionsList.append(o1)
+        OptionsList.append(o2)
+        OptionsList.append(o3)
+        OptionsList.append(o4)
+        OptionsList.append(o5)
+
+        # Randomize the order of country buttons, so that option 1 isn't always right ;)
+        random.shuffle(OptionsList)
+        for i in OptionsList:
+            view.add_item(i)
+
+        await ctx.send(files=[file, N['file']], embed=N['newEmbed'], view=(view if quota > 0 else None))
+        initial_score: int = 5000
+        scoreDecay: float = timer()
+        # Clean asset folder when done
+        shutil.rmtree('bot/cogs/geo_cog/tempAssets/')
+        os.mkdir('bot/cogs/geo_cog/tempAssets/')
+
+
+    @ext.command()
+    @commands.cooldown(1, 30, commands.BucketType.user)
+    @ext.example("?geoguess lb")
+    @ext.long_help("Displays the leaderboard")
+    @ext.short_help("leaderboard")
+    async def lb(self, ctx, *, member: discord.Member = None) -> None:
+        newEmbed: discord.Embed = discord.Embed(color=0x00FF61)
+        newEmbed.set_author(name="Discord Geoguessr by yeetusfeetus#9414",
+                            icon_url="https://cdn.discordapp.com/attachments/782728868179607603/1087563023243300884/authorimg.png")
+        LBoutput: str = ""
+
+        entries: int = 10 if (await self.repo.return_size())[0].get('COUNT(*)') >= 10 else (await self.repo.return_size())[0].get('COUNT(*)')
+        RankBTN: Button = Button(label="What's my rank?", style=discord.ButtonStyle.primary, row=1)
+
+        async def view_rank(interaction) -> None:
+            if await self.repo.get_best_preparation_for_member(interaction.user.id) is not None:
+                Size: int = (await self.repo.return_size())[0].get('COUNT(*)')
+                Rank: int = 0
+                for i in range(Size):
+                    if ( (await self.repo.get_rank())[i].get('user_id') == interaction.user.id ):
+                        Rank = (await self.repo.get_rank())[i].get('RANK')
+
+                Score: int = (await self.repo.sort_and_return())[ Rank - 1 ].get('score')
+                MSG: str = f'You are ranked **#{Rank}** and you have **{Score} points**'
+                await interaction.response.send_message(MSG, ephemeral=True)
+            else:
+                await interaction.response.send_message(f'You aren\'t even on the board, try a game out! Run ?geoguess game', ephemeral=True)
+
+        view = View(timeout=None)
+        RankBTN.callback = view_rank
+        view.add_item(RankBTN)
+
+        for i in range(entries):
+            LBoutput += f"**{i+1}.** <@!{(await self.repo.sort_and_return())[i].get('user_id')}> - **{(await self.repo.sort_and_return())[i].get('score')} points**\n"
+
+        newEmbed.add_field(name="Leaderboard", value=LBoutput, inline=False)
+        await ctx.send(embed=newEmbed, view=view)
+        time.sleep(2)
+
+async def setup(bot: SockBot):
+    await bot.add_cog(GeoGuessCOG(bot))
```

Comment:
add newline to the end of the file
```suggestion
    await bot.add_cog(GeoGuessCOG(bot))
    
```